command/jobs/base.go

```diff
@@ -129,19 +127,15 @@ func (f *FirehoseBase) watch(w WatchJobListFunc) {
 
 		// Iterate jobs and find events that have changed since last run
 		for _, job := range jobs {
-			if job.ModifyIndex <= f.lastChangeIndex {
+			if job.JobSummary.ModifyIndex <= f.lastChangeIndex {
 				continue
 			}
 
-			if job.ModifyIndex > newMax {
-				newMax = job.ModifyIndex
-			}
-
 			w(job)
 		}
 
 		// Update WaitIndex and Last Change Time for next iteration
 		q.WaitIndex = meta.LastIndex
-		f.lastChangeIndex = newMax
+		f.lastChangeIndex = meta.LastIndex
```

Comment:
this will always match the value from `meta.LastIndex`anyway, given the line 136 change would never ever encounter a higher value that `meta.LastIndex` in the response anyway